web/packages/teleport/src/features.tsx

```diff
@@ -556,15 +556,24 @@ export class FeatureClusters implements TeleportFeature {
     title: 'Clusters',
     path: cfg.routes.clusters,
     exact: false,
+    // TODO (avatus) if cloud/dashboard, make this go directly to the manage cluster page
```

Comment:
We can ignore dashboards completely, users can't manage their upgrade window there (this form was only shown for Cloud originally, and dashboards aren't "cloud" clusters).

The reason this feature has a `cfg.isDashboard` is the contacts. I think `hasAccess` should check for `flags.contacts` instead -- dashboard roles have contacts RO (dashboard-user) or RW (dashboard-admin).

Reply:
ok, ill make changes to have the desired effects
1: keep the `showInDashboard` line so that dashboard users can see clusters again (this makes sense for self hosted, and was a requested change)
2. remove the cluster config stuff to make this PR simpler, and just put the upgrade window form back into help & support.

before I do 2, can I ask who requested it to be moved in the first place, that way I'm not undoing work that moved it for a specific reason?

Reply:
TBH, I'd like to hear what the team thinks about this before moving with 2


>before I do 2, can I ask who requested it to be moved in the first place, that way I'm >not undoing work that moved it for a specific reason?

cc @roraback and @rishibarbhaya-design 
For context: during the Contacts work we introduced this `Manage Cluster` page and moved the Window Upgrade form to the new page. But we missed that some users don't have permission to access it, differently from the page this form was before (`Help & Support`)

Reply:
I've removed the code to solve 1 into its own PR. ill update this draft once we hear back from the UX team about where to move the form around to
https://github.com/gravitational/teleport/pull/52004